e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/PipelineRunFormAO.java

```diff
@@ -322,6 +356,39 @@ public PipelineRunFormAO resetClusterChildNodes () {
         return this;
     }
 
+    public PipelineRunFormAO setWorkersPriceType(final String priceType) {
+        click(WORKERS_PRICE_TYPE);
+        context().find(PipelineSelectors.visible(byClassName("ant-select-dropdown"))).find(byText(priceType))
+                .shouldBe(visible)
+                .click();
+        return this;
+    }
+
+    public PipelineRunFormAO enableHybridClusterSelect () {
+        $(byXpath(".//span[.='Enable Hybrid cluster']/preceding-sibling::span")).click();
+        return this;
+    }
+
+    public PipelineRunFormAO clusterEnableCheckboxSelect(String checkBox){
+        if (checkBox.equals("Enable GridEngine")
+                || checkBox.equals("Enable Apache Spark")
+                || checkBox.equals("Enable Slurm")) {
+            context()
+                .find(byXpath(
+                       String.format(".//span[.='%s']/preceding-sibling::span[@class='ant-checkbox']", checkBox)))
+                .click();
+        } else {
+            fail("Wrong checkbox name was selected");
+        }
+        return this;
+    }
+
+    public PipelineRunFormAO checkConfigureClusterLabel(String label) {
+        assertEquals(context().find(byXpath(".//div[@class='ant-row-flex ant-row-flex-end']/a"))
+                .shouldBe(visible).getText(), label);
```

Comment:
Better to use selenide checks - for example, `shouldHave(text(label))`. it will take a screenshot if it fails

Reply:
Done